docs/guides/device-gateway/sdk.md

```diff
@@ -0,0 +1,344 @@
+
+- `{HOST}` - the host where the API is running, leave blank and remove the colon for `localhost`
+- `{PORT}` - the port where the API is running
+- `{TUNNEL_DOMAIN}`- the domain you want to serve the API on the device or device gateway
+- `{YOUR_AGENT_DOMAIN}` - the domain you set on line `109` of the code
+
+```curl
+curl -k \
+-X POST \
+-H "Content-Type: application/json" \
+-d '{"protocol": "http", "forwards_to": "{HOST}:{PORT}", "domain": "{TUNNEL_DOMAIN}"}' \
+https://{YOUR_AGENT_DOMAIN}/tunnels
+```
+
+You should receive a `200` response similar to the following:
+
+```json
+{
+	"url": "https://device123.sitea.configurable-domain.com",
+	"protocol": "http",
+	"forwards_to": "localhost:8001",
+	"domain": "agent.sitea.configurable-domain.com",
+	"policy": null
+}
+```
+
+You'll notice that the `policy` property is `null`. This is because the request didn't include
+any [policies](https://ngrok.com/docs/http/traffic-policy/). However, this sample application applies the default polcies defined in `test.json`
+if no policies are included in the request.
+
+### List tunnels
+
+If you run the request to list tunnels again, you should receive a `200` response similar to the following. Notice that it includes the policies defined in `test.json`. You can update the policies
+in this file to the defaults you want applied to new tunnels when no policies are specified in
+the request:
+
+```json
+[
+	{
+		"url": "https://device123.sitea.configurable-domain.com",
+		"protocol": "http",
+		"forwards_to": "localhost:8001",
+		"domain": "device123.sitea.configurable-domain.com",
+		"policy": "{\"inbound\": [], \"outbound\": [{\"expressions\": [], \"name\": \"Add headers to requests\", \"actions\": [{\"type\": \"add-headers\", \"config\": {\"headers\": {\"is-ngrok\": \"0000000000\"}}}]}]}"
+	}
+]
+```
+
+### Start a tunnel with policies
+
+Run the following command, substituting the appropriate values for the variables to create a new tunnel, specifying [policies](https://ngrok.com/docs/http/traffic-policy/) directly in the request.
+
+```curl
+curl -k \
+-X POST \
+-H "Content-Type: application/json" \
+-d '{"protocol": "http", "forwards_to": "{HOST}:{PORT}","domain":"{YOUR_DOMAIN}", "policy": \
+{"inbound":[],"outbound":[{"expressions":[],"name":"Add headers to requests","actions":[{"type":"add-headers","config":{"headers":{"is-ngrok":"444"}}}]}]}}' \
+https://{YOUR_AGENT_DOMAIN}/tunnels
+```
+
+You should receive a `200` response similar to the following:
+
+```json
+{
+	"url": "https://device123.sitea.configurable-domain.com",
+	"protocol": "http",
+	"forwards_to": "localhost:8001",
+	"domain": "device123.sitea.configurable-domain.com",
+	"policy": {
+		"inbound": [],
+		"outbound": [
+			{
+				"expressions": [],
+				"name": "Add headers to requests",
+				"actions": [
+					{
+						"type": "add-headers",
+						"config": {
+							"headers": {
+								"is-ngrok": "444"
+							}
+						}
+					}
+				]
+			}
+		]
+	}
+}
+```
+
```

Comment:
Is it me, or did that indentation get crazy?

Reply:
No, it was wonky! Thank you!